PersistentCache/Src/SampleApplication/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) 2019 Alachisoft
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//    http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Alachisoft.NCache.Client;
+using Alachisoft.NCache.Runtime.Caching;
+using Microsoft.EntityFrameworkCore;
+using PersistenceProviders.Model;
+using PersistentNCache.DbProvider;
+using PersistentNCache.Model;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Test
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Name of cache to connect with
+            // Providers must be deployed with that cache
+            //Name of provider to be deployed
+            //PersistenceReadThru , PersistenceWriteThru and PersistenceStartupLoadewr
+            string cacheName = "persistentCache";
+            //Key for which the item is to be added
+            string key = "k1";
```

Comment:
key must be more specfic not generic  such as ProductL1001 some thing like that 
use our sample of backing source and startup loader

Reply:
sample application is completly changed now